bluepyopt/ephys/simulators.py

```diff
@@ -22,17 +22,34 @@ def __init__(self):
 
         import neuron  # NOQA
 
+        self.dt = dt
+        self.cvode_active = cvode_active
+
     @property
     def neuron(self):
         """Return neuron module"""
         import neuron
         return neuron
 
-    def run(self, tstop=None, cvode_active=True):
+    def run(self, tstop=None, cvode_active=None, dt=None):
         """Run protocol"""
 
         self.neuron.h.tstop = tstop
+
+        if cvode_active is None: # use cvode_active of simulator
+            cvode_active = self.cvode_active
+
         self.neuron.h.cvode_active(1 if cvode_active else 0)
-        logger.debug('Running Neuron simulator %.6g ms', tstop)
+
+        if dt is None: # use dt of simulator
```

Comment:
I think we might want to raise an exception if the user specifies cvode_active AND dt. What do you think ?
Shouldn't we also get the default dt from Neuron, instead of hard coding it to 0.025 ?